test/test_query.py

```diff
@@ -123,6 +123,19 @@ def test_subq_without_path_filter(self):
         with self.g.session_scope():
             self.assertEqual(self.g.nodes(models.Foo)
                              .subq_without_path(
-                                 'tests',
-                                 [lambda q: q.ids('test')])
+                'tests',
+                [lambda q: q.ids('test')])
                              .count(), self.g.nodes(models.Foo).count())
+
+
+@pytest.mark.parametrize(
+    "col, vals, expected_count", [
+        ("studies", ["C1"], 1),
+        ("studies", ["P2"], 3),
+        ("studies", ["C1", "P1"], 2),
+])
+def test_props_in(pg_driver, samples_with_array, col, vals, expected_count):
+    # filter
+    with pg_driver.session_scope() as s:
+        r = pg_driver.nodes(models.Foo).prop_in(col, vals).count()
+        assert r == expected_count
```

Comment:
`ages` is an integer array property, so changing for of the tests data to say `ages` with vals [29, 45] raises errors as `has_any` expects a text array